integration-tests/tests/import-dockstore-workflow.js

```diff
@@ -19,7 +19,7 @@ const testImportDockstoreWorkflowFn = async ({ context, page }) => {
       await select(page, 'Select a workspace', workspaceName)
       await click(page, clickable({ text: 'Import' }))
       await findText(page, testWorkflowIdentifier)
-    })({ context })
+    })(options)
```

Comment:
This felt funky before and it feels a little more funky now. Is there a better way to write this that doesn't put `options` all the way down here?